eventdata/operations/indexing.json

```diff
@@ -115,6 +115,25 @@
   },
   "index_template_name": "elasticlogs_i_write"
 },
+{
+  "name": "create_elasticlogs_index",
+  "operation-type": "createindex",
+  "index_name": "elasticlogs",
+  "index_template_body": {
+    "template": "elasticlogs",
+    "settings": {
+      "index.refresh_interval": "5s",
+      "index.codec": "best_compression",
+      "index.number_of_replicas": {{ replica_count | default(0) }},
```

Comment:
Inconsistent naming? In the README for [rally-tracks](https://github.com/elastic/rally-tracks/tree/master/eventdata#parameters) it says `number_of_replicas` (I know that technically this is not the same track...)

Reply:
Will align this